File: backend/java-spring/src/main/java/com/unosquare/worldcup/model/City.java
package com.unosquare.worldcup.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "cities")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class City {

    @Id
    private String id;
    private String name;
    private String country;
    private Double latitude;
    private Double longitude;
    private String stadium;
    private Double accommodationPerNight;
}

File: backend/dotnet-webapi/src/WorldCup.Api/Models/City.cs
namespace WorldCup.Api.Models;

public class City
{
    public string Id { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string Country { get; set; } = string.Empty;
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string Stadium { get; set; } = string.Empty;
    public double AccommodationPerNight { get; set; } = 150.0;
}
